crates/bevy_core/src/float_ord.rs

```diff
@@ -1,4 +1,4 @@
-use crate::bytes::AsBytes;
+use bytemuck::bytes_of;
```

Comment:
```suggestion
use bevy_core::bytes_of;
```
Are we able to use the reexported `bytes_of` from `bevy_core`?

Reply:
this is bevy_core. I could use `crate::bytes_of` maybe, but this crate just has to use bytemuck directly anyway.